### PR TITLE
static analysis cleanups

### PR DESCRIPTION
**Pull request by dgryski**
A few cleanups found by some tools

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -958,7 +958,7 @@ func buildAndRun(pkgName string, config *compileopts.Config, stdout io.Writer, c
 	err = run(cmd, result)
 	if err != nil {
 		if ctx != nil && ctx.Err() == context.DeadlineExceeded {
-			stdout.Write([]byte(fmt.Sprintf("--- timeout of %s exceeded, terminating...\n", timeout)))
+			fmt.Fprintf(stdout, "--- timeout of %s exceeded, terminating...\n", timeout)
```
Comment (archie2x) [inline main.go:961]: I mean... why isn't that:
```go
fmt.Printf("--- timeout of %s exceeded, terminating...\n", timeout)
```
?

**Review by archie2x — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (dgryski) [reply main.go:961]: Because `stdout` in this case is only *sometimes* `os.Stdout`, and other times we wrap it in our own buffering code.

Comment (archie2x) [reply main.go:961]: Figured. Thanks!

**Comment by deadprogram** [comment]
Thank you for the cleanup @dgryski and to @archie2x for review. Now merging.

**Pull request merged**